Mathlib/Topology/MetricSpace/GromovHausdorff.lean

```diff
@@ -889,55 +883,55 @@ theorem totallyBounded {t : Set GHSpace} {C : ℝ} {u : ℕ → ℝ} {K : ℕ 
       -- introduce `i`, that codes both `x` and `Φ x` in `fin (N p) = fin (N q)`
       let i : ℕ := E p x
       have hip : i < N p := ((E p) x).2
-      have hiq : i < N q := by rwa [Npq] at hip 
+      have hiq : i < N q := by rwa [Npq] at hip
       have i' : i = (E q) (Ψ x) := by simp only [Equiv.apply_symm_apply, Fin.coe_cast]
       -- introduce `j`, that codes both `y` and `Φ y` in `fin (N p) = fin (N q)`
       let j : ℕ := E p y
       have hjp : j < N p := ((E p) y).2
-      have hjq : j < N q := by rwa [Npq] at hjp 
+      have hjq : j < N q := by rwa [Npq] at hjp
       have j' : j = (E q) (Ψ y) := by simp only [Equiv.apply_symm_apply, Fin.coe_cast]
       -- Express `dist x y` in terms of `F p`
       have Ap : ((F p).2 ⟨i, hip⟩ ⟨j, hjp⟩).1 = ⌊ε⁻¹ * dist x y⌋₊ :=
         calc
           ((F p).2 ⟨i, hip⟩ ⟨j, hjp⟩).1 = ((F p).2 ((E p) x) ((E p) y)).1 := by
-            congr <;> apply Fin.ext_iff.2 <;> rfl
-          _ = min M ⌊ε⁻¹ * dist x y⌋₊ := by simp only [F, (E p).symm_apply_apply]
+            congr -- <;> apply Fin.ext_iff.2 <;> rfl
```

Comment:
Exactly, but I wasn't sure about why it was so.
Is it too late to add a porting note once bors has entered the game?

Reply:
No, you can still do that. Because it was a `bors d+` instead of a merge instruction.

But I think in this case we can just remove the commented code.
```suggestion
            congr
```
If you agree, please merge this suggestion and instruct bors to merge.

bors d=AntoineChambert-Loir